server/command.go

```diff
@@ -253,6 +271,10 @@ func (p *Plugin) runDisconnectCommand(user *model.User) (string, error) {
 // runHelpCommand runs command to display help text.
 func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 	text := starterText + strings.ReplaceAll(helpText+"\n"+settingHelpText, "|", "`")
+	if p.API.HasPermissionTo(user.Id, model.PermissionManageSystem) {
```

Comment:
Nit to use `p.client` instead of `p.API`. I wonder if we can make a golint rule for this. It doesn't make a huge difference, but it helps with references to `AppError` and `error` to avoid weird concrete vs interface things there. This `HasPermissionTo` method returns only a boolean so it's not any difference here

Reply:
We can make a seperate for adding golint rule for not using p.API since it will include a lot changes through the code once the new lint rule is implemented